composer.json

```diff
@@ -40,7 +40,8 @@
     },
     "autoload": {
         "psr-4": {
-            "Contao\\PhpStan\\": "src/"
+            "Contao\\PhpStan\\": "src/",
+            "Contao\\CoreBundle\\Tests\\": "../../contao/core-bundle/tests/"
```

Comment:
Do we not have to remove the entry from `autoload-dev` then?

Reply:
I think we need `autoload-dev` for `contao/phpstan` itself in test environments.
My added line only will provide autoload information if you use `contao/phpstan` in your project. 
If you remove it, the tests for `contao/phpstan` will fail with the same error message as in contao/contao#627.

Reply:
I see. Then I guess the solution is fine. @aschempp WDYT?

Reply:
Will it work with `contao/contao` and `contao/core-bundle` installed?